### PR TITLE
update npm module

### PR DESCRIPTION
**Pull request by zorancuc**

### CODE REVIEW

File: migrations/5_deploy_child_contracts.js [hunk 1]
```diff
@@ -7,9 +7,11 @@ const ChildChain = artifacts.require('ChildChain')
 const MRC20 = artifacts.require('MRC20')
 
 module.exports = async function(deployer, network, accounts) {
-  if (deployer.network !== 'bor') {
-    return
-  }
+  // if (deployer.network !== 'bor') {
```
Comment (jdkanani) [inline migrations/5_deploy_child_contracts.js:10]: This comment is intentional?

**Review by jdkanani — CHANGES_REQUESTED** [review]

File: migrations/5_deploy_child_contracts.js (hunk 1 above)
Comment (zorancuc) [reply migrations/5_deploy_child_contracts.js:10]: Yeah. We wanted to deploy contracts on bordevnet. In this case deployer.network must be "bordevnet".
The commented code will return the function and migration code will not run. So I commented it.

If this commented code is needed part, then we can add one more condition.
if (deployer.network !== 'bor' && deployer.network !== 'bordevnet') {
   return
}

File: truffle-config.js [hunk 2]
```diff
@@ -32,6 +32,15 @@ module.exports = {
       network_id: '*', // match any network
       gasPrice: '0'
     },
+    bordevnet: {
+      provider: () =>
+        new HDWalletProvider(
+          MNEMONIC,
+          `http://3.94.19.25:9545`
```
Comment (imyourm8) [inline truffle-config.js:39]: what is it for? Why new network is used?

Comment (zorancuc) [reply truffle-config.js:39]: It was used for safemath deployment test.

**Review by jdkanani — APPROVED** [review]

**Pull request merged**